ideainspect.groovy

```diff
@@ -242,6 +261,59 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println "#"
```

Comment:
I only read the code and I'm refering to the `println` block: 
````
+  println "# idea-cli-inspector: GENERATED START"
+  println "#   IDEA currently does currently not allow to pass the desired inspection scope as program parameter"
+  println "#   but only as property. To reflect a command-line based inspection run this entry was updated/added."
+  println "#   NOTE: This property only affect command line based inspection runs"
````

I assume the text there only goes into the `STDOUT` of the script but not into the `idea.properties` file. It seems you iterate over the `outputLines` array to create a updated copy of that file.